src/components/SearchableMultiSelect/SearchableMultiSelect.jsx

```diff
@@ -380,7 +406,9 @@ const SearchableMultiSelect = createClass({
 					<CheckboxLabeled.Label>
 						{_.isString(optionProps.children)
 							? this.renderUnderlinedChildren(optionProps.children, searchText)
```

Comment:
can't we replace these with `<Underline />`?

Reply:
i'd like to, but it does break snapshots since the shallow rendering + class names are changed

Reply:
🤔 yeah, not sure if we should consider that a breaking change